openhands/core/config/cli_config.py

```diff
@@ -1,3 +1,10 @@
+# IMPORTANT: LEGACY V0 CODE
```

Comment:
I think this file can be removed all together? The CLI has been moved and the old one was supposed to have been removed all together